packages/@okta/i18n/package.json

```diff
@@ -1,33 +1,31 @@
 {
-  "_from": "@okta/i18n@54.0.0-beta.g5d8e734",
```

Comment:
doesn't seem right. how did you copy over those i18n files?

Reply:
This is the package.json from the npm module on my local hard drive

I copied the other files (.properties, .json) from the dist/ folder of the npm module on my local hard drive.